pages/multi/[address]/index.tsx

```diff
@@ -8,13 +8,18 @@ import HashView from "../../../components/dataViews/HashView";
 import MultisigHoldings from "../../../components/dataViews/MultisigHoldings";
 import MultisigMembers from "../../../components/dataViews/MultisigMembers";
 import DelegationForm from "../../../components/forms/DelegationForm";
+import ReDelegationForm from "../../../components/forms/ReDelegationForm";
+import RewardsForm from "../../../components/forms/RewardsForm";
 import TransactionForm from "../../../components/forms/TransactionForm";
+import UnDelegationForm from "../../../components/forms/UnDelegationForm";
 import Button from "../../../components/inputs/Button";
 import Page from "../../../components/layout/Page";
 import StackableContainer from "../../../components/layout/StackableContainer";
 import { useAppContext } from "../../../context/AppContext";
 import { getMultisigAccount } from "../../../lib/multisigHelpers";
 
+type TxView = "select" | "send" | "delegate" | "undelegate" | "redelegate" | "claimRewards";
```

Comment:
Very nice solution! What about chaning `"select"` to `null` to make it clear that this is a special case that works differently than the others?